### PR TITLE
docs(example): add twind example

### PR DESCRIPTION
**Pull request by nexxeln**
This example shows how to use [Twind](https://twind.dev) with Remix. 
Thank you. Remix rocks

demo gif

### CODE REVIEW

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Sweet! Thanks! Does this server render? It doesn't look like it does. Is it possible to make it so we don't get a flash of unstyled content before the JS loads?

**Comment by nexxeln** [comment]
Twind does support server side rendering: 
- https://twind.dev/usage-guides/ssr.html

I'll implement this in a bit.

**Comment by nexxeln** [comment]
Turns out, for SSR in Remix they have a package. I'll use that and commit the changes.

File: examples/twind/app/root.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <html lang="en" className="bg-[#121212] text-slate-100">
+      <head>
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        <Outlet />
+        <ScrollRestoration />
+        <Scripts />
+        {process.env.NODE_ENV === 'development' && <LiveReload />}
```
Comment (MichaelDeBoey) [inline examples/twind/app/root.tsx:33]:
```suggestion
        <LiveReload />
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Review by kentcdodds — APPROVED** [review]
Sweet! Thanks!

**Pull request merged**